src/index.jsx

```diff
@@ -0,0 +1,133 @@
+import React, {PureComponent} from 'react'
+
+import {create as createJSS, getDynamicStyles} from 'jss'
```

Comment:
I use strict camel case: createJss, it looks strange sometimes but its a better way.